From e40bcc11d89882046685a7b01d30c4599d3df91a Mon Sep 17 00:00:00 2001
From: Tianfei zhang <tianfei.zhang@intel.com>
Date: Mon, 8 Mar 2021 07:05:07 -0500
Subject: [PATCH 43/93] fpga: m10bmc-sec: Add type to struct m10bmc_sec

Add the enum fpga_sec_type type field to struct m10bmc_sec to facilitate
multiple MAX10 BMC secure update devices.

Signed-off-by: Tianfei zhang <tianfei.zhang@intel.com>
---
 drivers/fpga/intel-m10-bmc-sec-update.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/fpga/intel-m10-bmc-sec-update.c b/drivers/fpga/intel-m10-bmc-sec-update.c
index 92886883dd61..05f8d2db64ad 100644
--- a/drivers/fpga/intel-m10-bmc-sec-update.c
+++ b/drivers/fpga/intel-m10-bmc-sec-update.c
@@ -31,6 +31,7 @@ struct m10bmc_sec {
 	u32 fw_name_id;
 	bool cancel_request;
 	struct image_load *image_load;		/* terminated with { } member */
+	enum fpga_sec_type type;
 };
 
 struct image_load {
@@ -856,6 +857,7 @@ static int m10bmc_sec_probe(struct platform_device *pdev)
 
 	sec->dev = &pdev->dev;
 	sec->m10bmc = dev_get_drvdata(pdev->dev.parent);
+	sec->type = type;
 
 	if (type == N3000BMC_SEC)
 		sec->image_load = n3000_image_load_hndlrs;
-- 
2.39.0

